card/lib/card/subcards.rb

```diff
@@ -236,11 +236,20 @@ def extract_subcard_args! name, args
       subcards
     end
 
+    # ensure a leading '+'
+    def normalize_subfield_key key
+      if key.is_a?(Symbol) && Card::Codename.exist?(key)
+        key = Card::Codename.name(key)
+      end
+      key = "+#{key}" unless key.to_s.start_with? "+"
```

Comment:
`key.to_name.prepend_joint` doesn't require the conditional and avoids hard-coded plusses.

A shorter name would be better, since this is common.  `to_field` maybe?